lib/util/mask.ts

```diff
@@ -5,5 +5,5 @@ export function maskToken(str?: string): string {
         new Array(str.length - 3).join('*'),
         str.slice(-2),
       ].join('')
-    : str;
+    : '';
```

Comment:
Are you sure this won't change the behavior?

Reply:
I think in worst case it will alter output from `'undefined'` to `''`, and actually I would like to even make it `'<empty>'` or something like this

Reply:
@rarkins WDYT?

Reply:
I think it's fine